start-miner.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env sh
 
 . ./dbus-wait.sh
+. ./miner-gc.sh
```

Comment:
No, this is wrong. There's no need to do this. We either do it in `docker-compose.yml` or we do it in the [Dockerfile](https://github.com/NebraLtd/hm-miner/blob/master/Dockerfile). No point in reinventing the wheel here.